helm/flink-kubernetes-operator/templates/webhook.yaml

```diff
@@ -109,4 +109,36 @@ webhooks:
         operator: In
         values: [{{- range .Values.watchNamespaces }}{{ . | quote }},{{- end}}]
   {{- end }}
-  {{- end }}
+---
```

Comment:
I think we should have a separate config flag to enable this. We should handle the validating and mutating webhooks independently of each other as some envs might want one but not the other.